components/conversations/conversation.routes.js

```diff
@@ -0,0 +1,19 @@
+import { Router } from "express";
+import Conversation from "./conversation.model.js";
+
+const conversationRouter = Router();
+
+conversationRouter.get("/conversations", async (req, res) => {
```

Comment:
i would recommend to put the logic in seperated file called  yourModel.controller.js, because it can get messy , it will be like :

import yourModelController from 'yourModel.controller.js'
conversationRouter.get("/conversations",yourModelController.functionName)